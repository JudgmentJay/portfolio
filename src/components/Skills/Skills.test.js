import React from 'react'
import renderer from 'react-test-renderer'

import { Skills } from '../'

it('Renders correctly', () => {
	const tree = renderer
		.create(<Skills />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
