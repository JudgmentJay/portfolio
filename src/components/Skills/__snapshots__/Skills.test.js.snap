// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<section
  className="section"
  id="skills"
>
  <div
    className="container"
  >
    <h3
      className="sectionHeader"
    >
      Technical Skills
    </h3>
    <p
      className="intro"
    >
      Below is a summary of the various languages, technology, and software I am able to utilize and my estimated proficiencly levels for each.
    </p>
    <div
      className="primarySkills"
    >
      <div
        className="skill"
      >
        <span
          className="skillName"
        >
          html
        </span>
        <div
          className="bar"
        >
          <div
            className="fill"
            style={
              {
                "width": "25%",
              }
            }
          >
            95
            %
          </div>
        </div>
      </div>
      <div
        className="skill"
      >
        <span
          className="skillName"
        >
          css
        </span>
        <div
          className="bar"
        >
          <div
            className="fill"
            style={
              {
                "width": "25%",
              }
            }
          >
            95
            %
          </div>
        </div>
      </div>
      <div
        className="skill"
      >
        <span
          className="skillName"
        >
          react
        </span>
        <div
          className="bar"
        >
          <div
            className="fill"
            style={
              {
                "width": "25%",
              }
            }
          >
            85
            %
          </div>
        </div>
      </div>
      <div
        className="skill"
      >
        <span
          className="skillName"
        >
          javascript
        </span>
        <div
          className="bar"
        >
          <div
            className="fill"
            style={
              {
                "width": "25%",
              }
            }
          >
            80
            %
          </div>
        </div>
      </div>
    </div>
    <div
      className="extras"
    >
      <div
        className="secondarySkills"
      >
        <h4
          className="header"
        >
          Secondary Skills
        </h4>
        <div
          className="skill"
        >
          <span
            className="skillName skillName--small"
          >
            sass
          </span>
          <div
            className="bar"
          >
            <div
              className="fill"
              style={
                {
                  "width": "25%",
                }
              }
            >
              80
              %
            </div>
          </div>
        </div>
        <div
          className="skill"
        >
          <span
            className="skillName skillName--small"
          >
            node/express
          </span>
          <div
            className="bar"
          >
            <div
              className="fill"
              style={
                {
                  "width": "25%",
                }
              }
            >
              70
              %
            </div>
          </div>
        </div>
        <div
          className="skill"
        >
          <span
            className="skillName skillName--small"
          >
            email development
          </span>
          <div
            className="bar"
          >
            <div
              className="fill"
              style={
                {
                  "width": "25%",
                }
              }
            >
              70
              %
            </div>
          </div>
        </div>
        <div
          className="skill"
        >
          <span
            className="skillName skillName--small"
          >
            graphql
          </span>
          <div
            className="bar"
          >
            <div
              className="fill"
              style={
                {
                  "width": "25%",
                }
              }
            >
              60
              %
            </div>
          </div>
        </div>
      </div>
      <div
        className="strengths"
      >
        <h4
          className="header"
        >
          Strengths
        </h4>
        <ul>
          <li>
            Writing efficient, modular, reusable code
          </li>
          <li>
            High productivity/bandwidth
          </li>
          <li>
            Attention to detail
          </li>
          <li>
            Adhering to industry/team standards
          </li>
          <li>
            Finding ways to optimize performance
          </li>
          <li>
            Avoiding/fixing cross-browser compatibility issues
          </li>
          <li>
            Staying up to date with industry trends
          </li>
          <li>
            Passionate about learning new things
          </li>
        </ul>
      </div>
    </div>
  </div>
</section>
`;
